accounts-db/src/tiered_storage/writer.rs

```diff
@@ -30,6 +46,25 @@ impl<'format> TieredStorageWriter<'format> {
         })
     }
 
+    /// Persists a single account to a dedicated new account block and
+    /// return its stored size.
+    ///
+    /// The function currently only supports HotAccountMeta, and will
+    /// be extended to cover more TieredAccountMeta in future PRs.
+    fn write_single_account<T: ReadableAccount + Sync>(
```

Comment:
Yes, but since they share some other stuff in common, the end-look of this function will include a generic type of TieredAccountMeta, something like:

`fn write_single_account<T: TieredAccountMeta, U: ReadableAccount + Sync>(`

Reply:
I'm removing `write_single_account` out from this PR for now.  Let's revisit this question in the next PR when we have more context.

Reply:
Done!

Reply:
Let's discuss this in our Zoom meeting tomorrow.  We should be able to exchange ideas and come up with a good solution quicker.